pyRserve/rparser.py

```diff
@@ -338,6 +338,15 @@ def xt_array_bool(self, lexeme):
         numBools = self.__unpack(XT_INT, 1)[0]
         # read the actual boolean values, including padding bytes:
         raw = self.read(lexeme.dataLength - 4)
+        # Check if the array contains any NA values (encoded as \x02).
+        # If so we need to convert the 2's to None's and use a numpy
+        # array of type Object otherwise numpy will cast the None's into False's. 
+        # This is handled for us for numeric types since numpy can use it's own
+        # nan type, but here we need to help it out.
+        if 2 in raw:
+            data = numpy.frombuffer(raw[:numBools], dtype=numpy.int8).astype(numpy.object)
```

Comment:
`numpy.object` has been deprecated, would you mind to use plain `object` instead?